index.html

```diff
@@ -25,6 +29,15 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+
+      <nav id="nav-mobile">
+        <ul>
+          <li><a class="nav-item" href="#projects">Portoflio</a></li>
```

Comment:
The hamburger and close icon work great on mobile! But could you kindly correct this typographical error? I know you intended for this link to be `Portfolio` instead? Don't worry, happens to the best of us. 🤝

Reply:
Thanks a lot for the observation